packages/faultstat.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Faultstat < Package
+  description 'Faultstat measures page fault activity and swap utilization of processes'
+  homepage 'http://kernel.ubuntu.com/~cking/faultstat/'
```

Comment:
`https://kernel.ubuntu.com/~cking/faultstat/`?

Reply:
Changed to HTTPS.